### DISCUSSION TITLE
Project Sprint のリリース時にSlackに投稿していた分をdiscussionのどこかに移管したい

### DISCUSSION THREAD

**Discussion by motoi** [copilot-jp/project-sprint#392] [Development]
とりあえずAnnouncementsでよいのかな？
@KokoroKagawa どうでっしゃろ。

**Comment by KokoroKagawa** [comment]
Announcementsでよいと思います（わざわざ新チャンネル作るほどではないかと）。
リリース作成時に、当該リリースからdiscussionを作成する機能があるので、作業自体も簡単そう。
まずはv3.3.1を作成してみます。

> Create a discussion for this release
People will be able to leave comments and reactions on this release using Discussions.

**Reply by motoi** [reply]
これでよいとおもいました！

**Comment by KokoroKagawa** [comment]
Issue化したのでそちらで進めます！
